.travis.yml

```diff
@@ -40,16 +39,16 @@ matrix:
       - ./build/mvn test $MVN_ARGS -pl kyuubi-common,kyuubi-zookeeper,kyuubi-ha,kyuubi-ctl,kyuubi-metrics,kyuubi-hive-beeline,kyuubi-hive-jdbc,extensions/server/kyuubi-server-plugin -am
   - name: Build Kyuubi Flink on Linux ARM64
     script:
-      - ./build/mvn test $MVN_ARGS -pl externals/kyuubi-flink-sql-engine,kyuubi-server
+      - ./build/mvn test $MVN_ARGS -pl externals/kyuubi-flink-sql-engine,kyuubi-server,integration-tests/kyuubi-flink-it -DwildcardSuites=org.apache.kyuubi.it.flink.operation
   - name: Build Kyuubi Spark on Linux ARM64
     script:
-      - ./build/mvn test $MVN_ARGS -pl externals/kyuubi-spark-sql-engine,kyuubi-server
+      - ./build/mvn test $MVN_ARGS -pl externals/kyuubi-spark-sql-engine,kyuubi-server -DwildcardSuites=org.apache.kyuubi.operation.KyuubiOperationPerUserSuite
   - name: Build Kyuubi Trino on Linux ARM64
     script:
-      - ./build/mvn test $MVN_ARGS -pl externals/kyuubi-trino-engine,kyuubi-server
+      - ./build/mvn test $MVN_ARGS -pl externals/kyuubi-trino-engine,kyuubi-server -DwildcardSuites=org.apache.kyuubi.engine.trino
```

Comment:
This reminds me that we missed the trino it. We only have the trino engine test without Kyuubi server. isn't it ? @hddong